fairroot/base/sim/FairMCApplication.h

```diff
@@ -320,7 +326,7 @@ class FairMCApplication : public TVirtualMCApplication
     Bool_t fSaveCurrentEvent;
 
     /** Current state */
-    FairMCApplicationState fState;   //!
+    FairMCApplicationState fState {FairMCApplicationState::kPreInit};   //!
```

Comment:
Please fix the clang-format thing.

```suggestion
    FairMCApplicationState fState{FairMCApplicationState::kPreInit};   //!
```

Reply:
I have fixed the format.
As for squashing, I feel that different commits touch various things, so I do not see simple ways of squashing them.